modules/Core/pages/terms.php

```diff
@@ -17,12 +17,13 @@
 // Retrieve terms from database
 $site_terms = DB::getInstance()->get('privacy_terms', ['name', 'terms'])->results();
 if (!count($site_terms)) {
-    $site_terms = DB::getInstance()->get('settings', ['name', 't_and_c_site'])->results();
+    $site_terms = Util::getSetting('t_and_c_site');
+} else {
+    $site_terms = $site_terms[0]->value;
 }
-$site_terms = Output::getPurified($site_terms[0]->value);
+$site_terms = Output::getPurified($site_terms);
 
-$nameless_terms = DB::getInstance()->get('settings', ['name', 't_and_c'])->results();
-$nameless_terms = Output::getPurified($nameless_terms[0]->value);
+$nameless_terms = Util::getSetting('t_and_c');
```

Comment:
Could maybe do with keeping `Output::getPurified` here

Reply:
I don't know if I had a good reason for removing it or if I did it by accident.. in any case I'll restore getPurified